platform/android/export/export.cpp

```diff
@@ -1442,8 +1443,10 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		//printf("end\n");
 	}
 
-	void _process_launcher_icons(const String &p_processing_file_name, const Ref<Image> &p_source_image, const LauncherIcon p_icon, Vector<uint8_t> &p_data) {
-		if (p_processing_file_name == p_icon.export_path) {
```

Comment:
Move this check to the caller instead; this will simplify the method signature and make the logic simpler to understand.
Remove the `const LauncherIcon p_icon` parameter and pass the dimension instead. Those two changes will turn this method into a simple image processing method and make it simpler to use.